static/build.html

```diff
@@ -669,8 +664,10 @@ <h3 id="browser-and-webview">
 
             <pre>fetch --nohooks android</pre>
 
-            <p>Sync to the latest stable release for Android (replace $VERSION with the correct
-            value):</p>
+            <p>You can obtain the proper configuration from the <a href="https://github.com/GrapheneOS/Vanadium">
```

Comment:
You should probably move this down a bit further if you want to move it like this. I don't think it makes more sense here. It should go with the configuration step.

Reply:
Thanks, let me review.